lang/ja.json

```
@@ -467,7 +467,7 @@
         "TrackingScopeVisit": "ビジット",
         "TransitionsRowActionTooltip": "このページを見る前と後にビジターが何をしたかを見る",
         "TransitionsRowActionTooltipTitle": "トランジションを開く",
-        "TranslatorName": "Takafumi/Drupal Japan - http://drupal.jp Takeshi Ueda/Matomo Japan Team - http://piwikjapan.org",
+        "TranslatorName": "Takafumi/Drupal Japan - http://drupal.jp Takeshi Ueda/Matomo Japan Team - http://matomo.jp",
```

We should also change to https as it's available.
However the drupal.jp domain is not working, should we remove it then?
```suggestion
        "TranslatorName": "Takafumi/Drupal Japan - http://drupal.jp Takeshi Ueda/Matomo Japan Team - https://matomo.jp",
```

---

Any complains with

"Takafumi/Drupal Japan, Takeshi Ueda/Matomo Japan Team - https://matomo.jp"

---

that's probably how I'd do it 👍

---

or "Takafumi/Drupal Japan, Takeshi Ueda/Matomo Japan Team (https://matomo.jp/)", so that it's more obvious the domain is just for the second person.